src/io/flutter/sdk/FlutterCommand.java

```diff
@@ -48,13 +48,28 @@
   @NotNull
   private final List<String> args;
 
+  // TODO(github.com/flutter/flutter-intellij/issues/3293) This is a temporary "<pub-cache>/bin/webdev" that can be provided to test some
```

Comment:
please update these to bugs related to the problem being solved with the workaround not the feature being implemented.
its fine if the bug descriptions are short but that will make more sense to users viewing the TODO. For example, the bug you linked to will be closed but the TODOs will still exist. In general the bugs for the TODO should not be closed until the TODO is removed.

Reply:
Will do, I suppose it will make it more explicit, right now the issue, #3293, has a list of tasks associated with it... I will create some additional more explicit issues to track the specific tasks (4th bullet point, in this case.)

Reply:
Done, https://github.com/flutter/flutter-intellij/issues/3348